modules/client-configuration/pages/autoinst-reinstall.adoc

```diff
@@ -0,0 +1,14 @@
+[[reinstalling-registered-systems]]
+= Reinstalling Registered Systems
+
+To reinstall in place an already registered client, you first have to define an autoinstallation distribution and an autoinstallation profile. For details, see xref:client-configuration:autoinst-xxxxx.adoc[] and xref:client-configuration:autoinst-xxxxx.adoc[].
+
+Once this is done, navigate to menu:Systems[Systems List], select the client to reinstall, and select menu:Provisioning[Autoinstallation > Schedule] subtab.
+
+At this point, select the autoinstallation profile that you prepared, select a proxy if needed, and press btn:[Schedule Autoinstallation and Finish]. If your client is a traditional client, type [command]``mgr_check`` on that client.
+
+You may follow the progress of the installation in menu:Provisioning[Autoinstallation > Session Status], or simply on the client's screen. You will see the client reboot, and automatically select in the boot menu a new choice called [guimenu]``reinstall-system``.
```

Comment:
```suggestion
When you have defined the autoinstallation profile and distribution, you can perform the reinstallation.
Navigate to menu:Systems[Systems List], select the client to reinstall, and go to the menu:Provisioning[Autoinstallation > Schedule] subtab.
Select the autoinstallation profile that you prepared, select a proxy if needed, and click btn:[Schedule Autoinstallation and Finish]. 
If your client is a traditional client, type [command]``mgr_check`` at the command prompt on the client.
You can monitor progress of the installation by navigating to menu:Provisioning[Autoinstallation > Session Status], or on the client directly.
The client reboots, and in the boot menu selects a new choice called [guimenu]``reinstall-system``.
```

This is probably a procedure, not prose.

Reply:
Your prose is nice :smile_cat: , but I will indeed make it a procedure.